source/docs/v3/integrations/passport.blade.md

```diff
@@ -40,38 +40,58 @@ To use Passport inside the tenant part of your application, you may do the follo
 ```
 
 
-5. If you're using Passport 11.x, disable the automatic Passport route registering and register the routes manually by adding the following code to the `register` method in your `AppServiceProvider`:
+5. If you're using Passport 11.x, disable the automatic Passport route registering and register the routes manually by adding the following code to the `register` method in your `AuthServiceProvider`:
 
-    ```php
-    Passport::$registersRoutes = false;
+```php
+    Passport::ignoreRoutes();
 
     Route::group([
         'as' => 'passport.',
-        'middleware' => [InitializeTenancyByDomain::class], // Use tenancy initialization middleware of your choice
+        'middleware' => [
+            InitializeTenancyByDomain::class, // Use tenancy initialization middleware of your choice
+            PreventAccessFromCentralDomains::class,
+        ],
         'prefix' => config('passport.path', 'oauth'),
         'namespace' => 'Laravel\Passport\Http\Controllers',
     ], function () {
         $this->loadRoutesFrom(__DIR__ . "/../../vendor/laravel/passport/src/../routes/web.php");
     });
-    ```
+```
 
 6. Apply Passport migrations by running `php artisan tenants:migrate`.
 
 7. Set up [the encryption keys](#passport-encryption-keys).
 
 ## **Using Passport in both the tenant and the central application** {#using-passport-in-both-the-tenant-and-the-central-application}
-To use Passport in both the tenant and the central application, follow [the steps for using Passport in the tenant appliction](#using-passport-in-the-tenant-application-only) with the following adjustments:
+To use Passport in both the tenant and the central application:
 
-1. Copy the Passport migrations to the central application, so that the Passport migrations are in both the central and the tenant application.
-2. Remove `Passport::ignoreMigrations()` from the `register` method in your `AppServiceProvider` (if it is there).
-3. In your `AuthServiceProvider`'s `boot` method, add the `'universal'` middleware to the Passport routes, and remove the `PreventAccessFromCentralDomains::class` middleware (if it is there). The routes should look like this:
+1. Follow [the steps for using Passport in the tenant appliction](#using-passport-in-the-tenant-application-only).
+2. Copy the Passport migrations to the central application, so that the Passport migrations are in both the central and the tenant application.
+3. Remove `Passport::ignoreMigrations()` from the `register` method in your `AuthServiceProvider` (if it is there).
+4. In your `AuthServiceProvider`, add the `'universal'` middleware to the Passport routes, and remove the `PreventAccessFromCentralDomains::class` middleware (if it is there). The routes should look like this:
 ```php
+// Passport 10.x
 Passport::routes(null, ['middleware' => [
     'universal',
     InitializeTenancyByDomain::class
 ]]);
+
+// Passport 11.x
+Passport::ignoreRoutes();
```

Comment:
Do we call this inside the route file, or inside the SP?

Reply:
When I was testing this, I was calling this in the `AppServiceProvider`

Reply:
Right, I see what you mean now

Reply:
Updated the block https://github.com/stancl/tenancy-docs/pull/226/commits/3ec45aa2aeb1babec6f91da94973202058639f05

Reply:
Ah I see that previously it meant to say the same thing, but the combination of `The routes should look like` with the `Route::` codeblock made it *seem* like it belongs in the routes file. Wanted to ask why the code block wasn't changed in that PR but I see that it calls `loadRoutesFrom()`.

The latest change looks good, it's now clear that this is SP code.